model_compression_toolkit/common/visualization/final_config_visualizer.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Tuple, List
+from matplotlib import pyplot as plt
+from matplotlib.figure import Figure
+
+from model_compression_toolkit.common import Graph
+
+
+def get_layer_represent_name(node):
+    """
+    Returns the mapping between a layer's type and its name to appear in the visualization figure.
+
+    Args:
+        node: A graph node representing a model's layer.
+
+    Returns: The name of the layer to appear in the visualization.
+
+    """
+
+    return node.type.__name__
+
```

Comment:
Does it work for functional nodes as well? (does tf.add has it?)

Reply:
I'm not sure, I can check, but anyway we only apply it to nodes that have kernels that we want to quantize (we present the name only for the weights configurable nodes)

Reply:
If it's relevant only for weights configurable, so let's rename the function name to indicate it. Also, we need to check for example the case of tf.nn.conv2d (which is functional)